packages/client/bin/cli.ts

```diff
@@ -28,6 +28,8 @@ import {
   setLengthLeft,
   short,
 } from '@ethereumjs/util'
+//@ts-ignore trusted-setups doesn't include types
+import { trustedSetup as fast } from '@paulmillr/trusted-setups/fast.js'
```

Comment:
Nit: in tests it does not matter, but in production code I would suggest using a bit more descriptive name than `fast`. Something like `kzgSetupFast` or something.

Reply:
Ah, just using `trustedSetup` here should be fine :) 
```suggestion
import { trustedSetup } from '@paulmillr/trusted-setups/fast.js'
```